docs/guide/enabling-gpu-acceleration.md

```diff
@@ -1,19 +1,57 @@
 # Enabling GPU acceleration.
 
-Currently, GPU.js uses ES6 features and does not provide proper ES5 package. 
-Therefore, to allow the engine to run on browsers like Internet Explorer, it is not bundled with the engine.
+::: tip
+GPU acceleration is deprecated since [HyperFormula 1.2.0](../guide/release-notes.md#_1-2-0).
+:::
 
-In order to take advantage of the GPU acceleration, the GPU.js constructor has to be passed to the engine configuration 
-(supported version of GPU.js is 2.3.0):
+To enable GPU acceleration, follow the steps below.
 
-```javascript
+## Step 1: Use HyperFormula 1.1.0
+
+As `gpu.js` was removed from optional dependencies in [HyperFormula 1.2.0](../guide/release-notes.md#_1-2-0), you can only use GPU acceleration in HyperFormula 1.1.0 (or lower).
```

Comment:
@kirszenbaum 
Correct me if I'm wrong, @wojciechczerniak, but I think `gpu.js` support will still be provided in `1.2.0`, but the user will have to install `gpu.js` on their own.

Reply:
You're right 👍 For devs nothing changes. Flow is the same, `npm install gpu.js`, `import gpu from 'gpu.js'`, and give it to HF settings `{ gpu: gpu }`. The only change is that NPM didn't respected that last time.

Reply:
@jansiegel @wojciechczerniak - got it, is this OK then? 5a1de61